src/app/api/webhook-subscriber.test.ts

```
@@ -0,0 +1,429 @@
+        agentEventBus = new AgentEventBus();
+    });
+
+    afterEach(() => {
+        // Clean up webhook subscriber and abort controllers
+        webhookSubscriber.cleanup();
+
+        // Reset all mocks
+        vi.resetAllMocks();
+
+        // Clear the test environment
+        delete process.env.NODE_ENV;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)